cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -166,13 +169,55 @@
     this.twillSpec = spec;
     this.resourcePrefix = resourcePrefix;
     this.extraLabels = extraLabels;
+    dependentRunnableNames = new HashSet<>();
```

Comment:
Use `this.dependentRunnableNames` to be consistent with the other field assignments in the constructor.